Resources/Prototypes/secret_weights.yml

```diff
@@ -1,8 +1,30 @@
 - type: weightedRandom
   id: Secret
-  weights:
+  weights: #these are the presets!!! NOT gamerules.
     Nukeops: 0.20
     Traitor: 0.50
     Zombie: 0.10
     Revolutionary: 0.20
-    
\ No newline at end of file
+
+- type: weightedRandom
+  id: Secret_MRP
```

Comment:
SecretMRP, we don't use snake_casing for protoype IDs.

Reply:
In next commit